pallets/identity/src/auth.rs

```diff
@@ -59,6 +59,14 @@ impl<T: Config> Pallet<T> {
             number_of_given_auths < T::MaxGivenAuths::get(),
             Error::<T>::ExceededNumberOfGivenAuths
         );
+
+        if let AuthorizationData::PortfolioCustody(pid) = &authorization_data {
+            ensure!(
+                pid.kind != PortfolioKind::Default,
+                Error::<T>::DefaultPortfoliosCannotHaveCustodians
+            );
+        }
```

Comment:
These checks should be done when the authorization is accepted, otherwise this method would have a lot of checks for many different authorizations.  It does mean that an authorization can be created that can't be accepted.